test/unit/math/opencl/kernel_generator/calc_if_test.cpp

```diff
@@ -94,4 +91,17 @@ TEST(KernelGenerator, calc_if_colwise_min_test) {
   EXPECT_MATRIX_NEAR(correct, res, 1e-9);
 }
 
+TEST(KernelGenerator, calc_if_double) {
+  MatrixXd m(3, 2);
+  m << 1.1, 1.2, 1.3, 1.4, 1.5, 1.6;
+
+  matrix_cl<double> m1_cl(m);
+  matrix_cl<double> m2_cl = m1_cl + stan::math::calc_if<true>(4.0);
```

Comment:
I always intended `calc_if` to be used as the last operation before the assignment (not used in addition as you did here). I am not sure this will work in general (actually I am pretty sure it won't work for `calc_if<false>`), so I would prefer not to give anyone looking at tests wrong ideas. 

Actually I believe you only needed this to compile within `if(false)`, not actually execute. So this test should be removed.

Reply:
Yeah, it was only to make a signature compile. It never actually ran. Will remove the test.